ssl/statem/statem_srvr.c

```diff
@@ -3751,14 +3753,29 @@ int tls_construct_new_session_ticket(SSL *s, WPACKET *pkt)
     unsigned char iv[EVP_MAX_IV_LENGTH];
     unsigned char key_name[TLSEXT_KEYNAME_LENGTH];
     int iv_len;
+    unsigned char tick_nonce[TICKET_NONCE_SIZE];
     size_t macoffset, macendoffset;
     union {
         unsigned char age_add_c[sizeof(uint32_t)];
         uint32_t age_add;
     } age_add_u;
 
     if (SSL_IS_TLS13(s)) {
+        size_t i, hashlen;
+        uint64_t nonce;
+        const char nonce_label[] = "resumption";
```

Comment:
One of advantages of declaring literals const is that it's placed in read-only segment [so that attempt to write to it, as result of bug, should SEGV]. const without static creates [writable] string on stack. In other words const static would be more appropriate...